test/TestHelper.h

```diff
@@ -157,6 +157,9 @@ class ImportTest
 		eth::Network netId;
 	};
 	std::vector<transactionToExecute> m_transactions;
+	typedef std::pair<eth::State, eth::AccountMaskMap> stateAndMap;
```

Comment:
Please use `using` instead of `typedef`: `using stateAndMap = std::pair<...`. And name the types with capital letter.